test/sharness/t0085-pins.sh

```diff
@@ -160,6 +160,20 @@ test_pin_progress() {
   '
 }
 
+test_pin_verbose_progress() {
+  test_pin_dag_init
+
+  test_expect_success "'ipfs pin add --progress --verbose' file" '
+    ipfs pin add --progress --verbose $HASH 2> test_pin_verbose_progress_err
+  '
+
+  test_expect_success "pin progress reported correctly" '
+    cat test_pin_verbose_progress_err
+    grep -q "Pinning" test_pin_verbose_progress_err
```

Comment:
Can we test for the whole line here? I think we have the hash available in `$HASH`, is that true?

```suggestion
    grep -q "Pinning $HASH" test_pin_verbose_progress_err
```

Another thing that would be nice to have is a test that shows progress for recursively pinning a hash that has a root block and other blocks underneath it. I think that test can be approached the same as the test you have here. 

What do you think?

Reply:
Fixed with $HASH.

This test is kind of checks this but without explicit checks. Because the pin output is:
```
```